src/components/Dropdown/ControlledDropdown.tsx

```diff
@@ -25,6 +26,7 @@ const ControlledDropdown = forwardRef<
       maxPaneWidth = 270,
       isPaneElevated = false,
       hasPaneArrow = false,
+      focusTrap = false,
```

Comment:
Let's change this to `true` and then add `focusTrap=false` to the Tooltip component. Then we can merge this